README.md

```diff
@@ -36,5 +36,51 @@ For more detailed instructions to install Anaconda, check the [Windows](https://
 
 **Note:** When you are already familiar to the command line and Python environments you could opt to use Miniconda instead of Anaconda and download it  from <https://conda.io/miniconda.html>. The main difference is that Anaconda provides a graphical user interface (Anaconda navigator) and a whole lot of scientific packages (e.g <https://docs.anaconda.com/anaconda/packages/py3.6_win-64/>) when installing, whereas for Miniconda the user needs to install all packages using the command line. On the other hand, Miniconda requires less disc space. By choosing Miniconda, create the workshop environment using the `environment.yml` file: `conda env create -f environment.yml`
 
+### Install/check of required packages
+
+This tutorial will require recent installations of
+
+- [NumPy](http://www.numpy.org)
+- [SciPy](http://www.scipy.org)
+- [matplotlib](http://matplotlib.org)
+- [pandas](http://pandas.pydata.org)
+- [pillow](https://python-pillow.org)
+- [scikit-learn](http://scikit-learn.org/stable/)
+- [seaborn](http://seaborn.pydata.org/)
+- [IPython](http://ipython.readthedocs.org/en/stable/)
+- [Jupyter Notebook](http://jupyter.org)
+
+
+The last one is important and you should be able to type:
+
+```bash
+jupyter notebook
+```
+
+in your terminal window and see the notebook panel load in your web browser. Try opening and running a notebook from the material to see check that it works. Alternatively you can use Jupyter lab.
+
+After obtaining the material, we **strongly recommend** you to open and execute the Jupyter Notebook `jupter notebook check_env.ipynb` that is located at the top level of this repository. Inside the repository, you can open the notebook by executing
+
+```bash
+jupyter notebook check_env.ipynb
```

Comment:
Wouldn't it be simpler to make this a python file instead of notebook, so you can do `python check_env.py` and see the output?

Reply:
agreed. I just copied from the scipy but I think that it is easier. we can check for jupyter notebook then as well